react/components/SanitizedHTML.tsx

```diff
@@ -68,6 +68,13 @@ export const DEFAULTS = {
     'tr',
     'u',
     'ul',
+    // unfortunately some stores have these tags in product descriptions
+    'link',
```

Comment:
Why this was moved to this file?

Reply:
They don't use our `product-description` component: `exito-product-details-3-x-productDescriptionText`